test/pathod/test_utils.py

```diff
@@ -27,13 +29,8 @@ def test_data_path():
     tutils.raises(ValueError, utils.data.path, "nonexistent")
 
 
-def test_inner_repr():
-    assert utils.inner_repr("\x66") == "\x66"
-    assert utils.inner_repr(u"foo") == "foo"
-
-
 def test_escape_unprintables():
     s = "".join([chr(i) for i in range(255)])
-    e = utils.escape_unprintables(s)
+    e = utils.escape_unprintables(six.b(s))
```

Comment:
This probably works well, but we should adopt a strict "Python 3 by default" policy. That is, once we have full Python 3 compatibility, we should just be able to delete everything releated to `six`. How about this?

```python
s = bytes(range(256))
if six.PY2:
    s = "".join([chr(i) for i in range(255)])
```